core/src/main/java/org/mapfish/print/servlet/BaseMapServlet.java

```diff
@@ -117,4 +118,28 @@ protected final StringBuilder getBaseUrl(final HttpServletRequest httpServletReq
         }
         return baseURL;
     }
+
+    /**
+     * Set the cache duration for the queries that can be cached.
+     * @param cacheDurationInSeconds the duration
+     */
+    public final void setCacheDuration(final int cacheDurationInSeconds) {
+        this.cacheDurationInSeconds = cacheDurationInSeconds;
+    }
+
+    /**
+     * Disable caching of the response.
+     * @param response the response
+     */
+    protected void setNoCache(final HttpServletResponse response) {
+        response.setHeader("Cache-Control", "no-cache");
```

Comment:
NB: `no-cache` is a value usually found in requests, not responses. I'm not sure of the effect in this case.

To be on the safe side, I'd update this to read `private, max-age=0` instead.